plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -232,6 +268,13 @@
       - Specifies the VM name. Only used on the configuration web interface.
       - Required only for C(state=present).
     type: str
+  nameservers:
+    description:
+      - 'cloud-init: DNS server IP addresses.'
```

Comment:
Like `searchdomains`.
```suggestion
      - 'cloud-init: DNS server IP address(es).'
```